packages/ibm-products/src/components/Datagrid/Datagrid/addons/Filtering/OverflowCheckboxes.js

```diff
@@ -0,0 +1,78 @@
+import { Button, Checkbox, FormGroup } from '@carbon/react';
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+import { componentClass } from './FilterPanel';
+
+const OverflowCheckboxes = ({
+  components,
+  filtersState,
+  column,
+  setFiltersState,
+  type,
+  applyFilters,
+}) => {
+  const [showAllItems, setShowAllItems] = useState(false);
+  const firstFiveItems = filtersState[column].value.slice(0, 5);
+  const restOfTheItems = filtersState[column].value.slice(
+    5,
+    filtersState[column].value.length - 1
+  );
+
+  const renderCheckbox = (option) => (
+    <Checkbox
+      key={option.labelText}
```

Comment:
This is causing a bunch of console errors because all of the `dummyCheckboxes` have the same `labelText` which is causing them to all have the same key instead of a unique key.

Reply:
Fixed it by making the label text unique